public/app/features/panel/panellinks/link_srv.ts

```diff
@@ -75,7 +76,7 @@ const valueVars = [
 ];
 
 const buildLabelPath = (label: string) => {
-  return label.indexOf('.') > -1 ? `["${label}"]` : `.${label}`;
+  return label.indexOf('.') > -1 || label.trim().indexOf(' ') > -1 ? `["${label}"]` : `.${label}`;
```

Comment:
Nit for readability:
```suggestion
  return label.includes('.') || label.trim().includes(' ') ? `["${label}"]` : `.${label}`;
```